docs/develop/customize.md

```diff
@@ -22,3 +24,16 @@ Passing the `--no-install-js` option, skips re-installation of JS dependencies.
 Go to the browser [*https://localhost:5000/*](https://localhost:5000) or refresh the page. And voilà! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
```

Comment:
self note: there is no need to migrate this to ademonition, it has been done in the other PR waiting for merge.